qt/scientific_interfaces/ISISReflectometry/GUI/Common/Decoder.cpp

```diff
@@ -286,7 +287,7 @@ void Decoder::decodeRunsTable(QtRunsTableView *gui, ReductionJobs *redJobs, Runs
 
   if (m_projectSave) {
     // Apply styling and restore completed state for output range values
-    presenter->notifyRowOutputsChanged();
+    presenter->notifyRowModelChanged();
```

Comment:
Does `notifyRowModelChanged` need moving outside this `if` statement now? Previously this was only applicable when doing a save of the whole project (including output workspaces, hence the processed-state could be saved and restored), but I think this will also be needed to restore the index when loading a batch file. Although I'm not sure - does it currently work?

Reply:
It currently works as-is, loading a batch or a project keeps the lookup index how I'd expect them to be